### PR TITLE
add drop_role and set_default_role antlr visitor for MySQL DCLStatement

### PR DESCRIPTION
**Pull request by beijing-penguin**
Changes proposed in this pull request:
- add drop_role and set_default_role.xml
- add setDefaultRole role in MySQLStatement.g4

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java [hunk 1]
```diff
@@ -249,7 +250,17 @@ public ASTNode visitShowLike(final ShowLikeContext ctx) {
...
     }
-
+    
+    @Override
+    public ASTNode visitDropRole(DropRoleContext ctx) {
+        return new DCLStatement();
+    }
+    
+    @Override
+    public ASTNode visitSetDefaultRole(SetDefaultRoleContext ctx) {
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: Is it necessary to add? I could find this one in `META-INF/parsing-rule-definition/mysql/sql-statement-rule-definition.xml`

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/MySQLStatement.g4 [hunk 2]
```diff
@@ -45,6 +45,7 @@ execute
     | renameUser
     | createRole
     | dropRole
+    | setDefaultRole
```
Comment (tristaZero) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/MySQLStatement.g4]: Which rule g4 file does it exist?

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java [hunk 3]
```diff
@@ -26,6 +26,7 @@
 import org.antlr.v4.runtime.tree.TerminalNode;
 import org.apache.shardingsphere.sql.parser.api.SQLVisitor;
 import org.apache.shardingsphere.sql.parser.autogen.MySQLStatementBaseVisitor;
+import org.apache.shardingsphere.sql.parser.autogen.MySQLStatementParser.*;
```
Comment (SteNicholas) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: Please don't use * for import. Check the code style.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/drop_role.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-parser-test-cases>
+    <common sql-case-id="drop_role" />
```
Comment (SteNicholas) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/drop_role.xml]: Should this use drop-role not common？

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/set_default_role.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-parser-test-cases>
+    <common sql-case-id="set_none_role" />
```
Comment (SteNicholas) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/set_default_role.xml]: Similar to above.

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/MySQLStatement.g4 (hunk 2 above)
Comment (beijing-penguin) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/org/apache/shardingsphere/sql/parser/autogen/MySQLStatement.g4]: > Which rule g4 file does it exist?

DCLStatement.g4

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java (hunk 3 above)
Comment (beijing-penguin) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: ok，update!

File: shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/drop_role.xml (hunk 4 above)
Comment (beijing-penguin) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/resources/visitor/dcl/drop_role.xml]: run test unit is ok

File: shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java (hunk 1 above)
Comment (beijing-penguin) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java]: setDefaultRole have add to sql-statement-rule-definition.xml，please review again

**Pull request merged**